components/Cards/cards.css

```diff
@@ -0,0 +1,116 @@
+/* global css sheet for fonts and icons */
+@import "/css/main.css";
+
+/* main card component */
+.card {
+    height: 14rem;
+    width: 40rem;
+    border: 2px solid var(--universal-border-color);
+    margin: 0.5rem;
+    /* overflow: auto; */
+}
+
+/* image HTML5 tag */
+img {
+    height: 13rem;
+    width: 15rem;
+    border: 2px solid var(--universal-border-color);
+    margin: 0.3rem;
+}
+
+/* card content class */
+.card-content {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0.3rem;
+    margin-left: 0.2rem;
+}
+
+/* all important classes groped togther */
+.card-with-badge,
+.btn-icons,
+.card-dismiss,
+.card-with-text-overlay,
+.card-with-text-only,
+.card-horizontal {
+    display: flex;
+}
+
+/* HTML5 btton element */
+button {
+    height: 2rem;
+    margin: 0.3rem 0.3rem 0 0;
+    display: flex;
+    align-items: center;
+    border-radius: 0.3rem;
+    cursor: pointer;
+}
```

Comment:
Also use your button component here why to declare another css when you already have it